cmd/root_cmd.go

```diff
@@ -23,7 +26,7 @@ var rootCmd = cobra.Command{
 func RootCommand() *cobra.Command {
 	rootCmd.AddCommand(&serveCmd, &migrateCmd, &versionCmd, adminCmd())
 	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "the config file to use")
-
+	rootCmd.PersistentFlags().StringVarP(&watchDir, "watch-dir", "w", "", "config directory to watch for changes")
```

Comment:
Maybe to be more explicit:

```suggestion
	rootCmd.PersistentFlags().StringVarP(&watchDir, "watch-config-dir", "w", "", "config directory to watch for changes")
```